addons/website/static/src/components/fullscreen_indication/fullscreen_indication.js

```diff
@@ -0,0 +1,23 @@
+/** @odoo-module **/
+
+import { useEffect } from '@web/core/utils/hooks';
+
+const { Component, tags } = owl;
+
+export class FullscreenIndication extends Component {
```

Comment:
Note: now that is it is done, it's cool, it had to be done ultimately but really I imagined that kind of feature being implemented last in this refactoring :wink: Focus on main stuff :wink: FP & AL have to see things moving to be reassured and if I show them the ESC button feature in the current POC... they won't be impressed :smile: